src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -98,6 +98,34 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
         return Promise.reject(new OrderFinalizationNotRequiredError());
     }
 
+    protected getGooglePayOptions(): GooglePayPaymentInitializeOptions {
```

Comment:
instead of creating this methods, could you instead define these variables as protected?

Reply:
methods removed, changed superclass attributes to protected